packages/firestore/test/integration/bootstrap.ts

```diff
@@ -24,6 +24,8 @@ import '../../index';
  * https://github.com/webpack-contrib/karma-webpack#alternative-usage
  */
 
+process.env.TEST_PLATFORM = 'browser';
```

Comment:
Why do you need to set it if if it is the browser test? `TEST_PLATFORM` is only used for node test, right?

Reply:
Yes... I had to change the webpack bundler to allow me to inject browser-lite. With that change, we are now using the new networking stack for the Lite test. The change in this file was meant to do this, but was 100% wrong (but with the best of intentions!).